mapping/sustainability.yaml

```
@@ -430,7 +469,7 @@
     title: Climate and disaster risk assessment
     module: Environment and climate
     indicator: Climate and disaster risk
-    disclosure format: Clarify the type of climate and disaster risks to which the project is exposed to (E.g. \[document or free text to list and explain risks\]). This tends to be part of the appraisal documents.
+    disclosure format: Clarify the type of climate and disaster risks to which the project is exposed to. This tends to be part of the appraisal documents.
```

```suggestion
    disclosure format: Clarify the type of climate and disaster risks to which the project is exposed. This tends to be part of the appraisal documents.
```
The last 'to' is redundant. @mgraca-prado Would you like to make the change in the data point?

---

I've just removed this last 'to' to make the sentence more grammatically correct.